plugins/org.jboss.tools.openshift.core/META-INF/MANIFEST.MF

```diff
@@ -32,6 +32,7 @@ Export-Package: org.jboss.tools.openshift.core;x-friends:="org.jboss.tools.opens
  org.jboss.tools.openshift.core.connection;x-friends:="org.jboss.tools.openshift.test,org.jboss.tools.openshift.ui",
  org.jboss.tools.openshift.core.preferences;x-friends:="org.jboss.tools.openshift.test,org.jboss.tools.openshift.ui",
  org.jboss.tools.openshift.core.server;x-friends:="org.jboss.tools.openshift.test,org.jboss.tools.openshift.ui",
+ org.jboss.tools.openshift.core.util,
```

Comment:
thanks for spotting this. I used a Quick-Fix to include this package in the 'exports' but I did not realize that the 'x-friends' option was missing.